en/docs/learn/api-microgateway/deploy-apis-in-privatejet-mode.md

```diff
@@ -0,0 +1,321 @@
+# PrivateJet mode for Microgateways
+With many of applications gearing towards microservice architecture, it’s no surprise that container-orchestration systems such as Kubernetes have become so popular with functionalities such as automating computer application deployment, scaling, and management, 
```

Comment:
Shall we rephrase it as " In an age where more and more applications are adopting the microservice architecture, it is obvious that container-orchestration systems such as Kubernetes have gained popularity due to the attractive functionalities they offer to simplify a number of complex management tasks? Automating computer application deployment, scaling, and management are a few of such functionalities to name.

Reply:
done